frontend/src/context/analysisResultStateSlice.ts

```diff
@@ -408,16 +530,29 @@ export const requestAndStoreExposedPopulation = createAsyncThunk<
     // TODO - use raster legend title
     const legendText = wfsLayer ? wfsLayer.title : 'Exposure Analysis';
 
-    const tableRows: TableRow[] = generateTableFromApiData(
+    const tableColumns = generateTableColumnsFromApiData(
+      featuresWithBoundaryProps,
+      key,
+    );
+
+    let tableRows: TableRow[] = generateTableFromApiData(
       [statistic],
       featuresWithBoundaryProps,
       adminBoundariesData,
       apiRequest.group_by,
       null,
       [], // no extra columns
       true,
+      key,
     );
 
+    if (key) {
```

Comment:
This might not be the best test to decide if we should merge rows or not. Let's think about a few other options to make this clearer

Reply:
From the testing I've made to some countries, there is a pattern that when a key is present from the `exposure` property inside the `params` then we are reffering to tropical storms. However we could use a more unique key for this. However I am not realy sure what exactly though.